index.js

```diff
@@ -11,6 +14,22 @@ server.get('/helloworld', (req, res) => {
     res.send('Hello World - first initial dev')
 })
 
+// Endpoint for submitting a json request
+server.post('/inspecturls', jsonParser, (req, res) => {
+    
+    // Convert boduy to JSON
+    var inspector = req.body;
+    console.log("JSON input parsed into an object");
+
+    // Output to console window for now, we will do something different with it later
+    console.log(JSON.stringify(inspector));
+    res.send('JSON should have been output to console window');
+
+    inspector.urls.forEach(element => {

```

Comment:
Quick comment above this just to say why for now

Reply:
Done